layouts/default.vue

```diff
@@ -1,5 +1,8 @@
 <template>
   <main>
+    <!--[if lte IE 12]>
+    <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
+    <![endif]-->
```

Comment:
As per our DoD, shouldn't we just add a new item to the backlog to help people navigate without the globe?

Reply:
I'll add the item.